src/java/main/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -233,25 +233,25 @@ public QuorumCnxManager(QuorumPeer self) {
         listener = new Listener();
         listener.setName("QuorumPeerListener");
     }
-
+    
```

Comment:
style changes are great but would it be possible to separate them from changes to functionality? that way it becomes easier to understand changes that are made when going through the version control log